app/Locale/pt_PT/translations.php

```diff
@@ -1306,9 +1306,9 @@
     'Sign-up' => 'Registe-se',
     'Credentials' => 'Credenciais',
     'New user' => 'Novo utilizador',
-    'This username is already taken' => 'Este nome de usuário já foi utilizado',
-    // 'A link to reset your password has been sent by email.' => '',
-    // 'Your profile must have a valid email address.' => '',
-    // 'Unfortunately, we are unable to reset your password. Did you entered a valid username? Do you have an email address in your profile?' => '',
-    // 'TRL - Turkish Lira' => '',
+    'This username is already taken' => 'Este nome de usuário já está a ser utilizado',
```

Comment:
Fell free to change "usuário" to "utilizador" that might got forgotten by me on my last translation.